core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -1745,7 +1745,7 @@ class KafkaConfig private(doLog: Boolean, val props: java.util.Map[_, _], dynami
   val deleteRecordsPurgatoryPurgeIntervalRequests = getInt(KafkaConfig.DeleteRecordsPurgatoryPurgeIntervalRequestsProp)
   val autoLeaderRebalanceEnable = getBoolean(KafkaConfig.AutoLeaderRebalanceEnableProp)
   val leaderImbalancePerBrokerPercentage = getInt(KafkaConfig.LeaderImbalancePerBrokerPercentageProp)
-  val leaderImbalanceCheckIntervalSeconds = getLong(KafkaConfig.LeaderImbalanceCheckIntervalSecondsProp)
+  val leaderImbalanceCheckIntervalSeconds: Long = getLong(KafkaConfig.LeaderImbalanceCheckIntervalSecondsProp)
```

Comment:
For leader.imbalance.per.broker.percentage, do we plan to support it in KRaft later or do we plan to deprecate it from both ZK and KRaft based controller?

Reply:
We don't have any plans to implement it for KRaft in the near future. I am in favor of deprecating the configuration properties for both implementation. Let me start a KIP discussion regarding this change.